src/cli/src/export.rs

```diff
@@ -101,10 +107,51 @@ pub struct ExportCommand {
     /// Disable proxy server, if set, will not use any proxy.
     #[clap(long)]
     no_proxy: bool,
+
+    /// if export data to s3
+    #[clap(long)]
+    s3: bool,
+
+    /// The s3 bucket name
+    /// if s3 is set, this is required
+    #[clap(long)]
+    s3_bucket: Option<String>,
+
+    /// The s3 endpoint
+    /// if s3 is set, this is required
+    #[clap(long)]
+    s3_endpoint: Option<String>,
+
+    /// The s3 access key
+    /// if s3 is set, this is required
+    #[clap(long)]
+    s3_access_key: Option<String>,
+
+    /// The s3 secret key
+    /// if s3 is set, this is required
+    #[clap(long)]
+    s3_secret_key: Option<String>,
+
+    /// The s3 region
+    /// if s3 is set, this is required
+    #[clap(long)]
+    s3_region: Option<String>,
```

Comment:
Can we offer a configuration that supports multiple types of object storage, like GreptimeDB does currently?

https://github.com/GreptimeTeam/greptimedb/blob/dee76f0a73c2252675b7837bedd695b47bc9ac58/src/datanode/src/config.rs#L44

Reply:
maybe since `export` is cli I wonder we offer cli command is needed

Reply:
I think it's necessary. For instance, if GreptimeDB stores data on `oss`, backups may also need to be stored on `oss`.

Reply:
got your point, follow the greptimedb config is fine? if so will try to use that way.
if that, seems we need to change the config logic

Reply:
Yes, I think following the GreptimeDB config is fine.